spec/unit/navigation/navigation.spec.js

```diff
@@ -66,6 +67,39 @@ describe('navigation toggle', function () {
     assert.equal(isVisible(nav), false);
   });
 
+  it('focuses the close button when the menu button is clicked', function () {
+    menuButton.click();
+    assert.equal(document.activeElement, closeButton);
+  });
+
+  it('focuses the menu button when the close button is clicked', function () {
+    menuButton.click();
+    closeButton.click();
+    assert.equal(document.activeElement, menuButton);
+  });
+
+  it('collapses nav if needed on window resize', function () {
+    menuButton.click();
+    sinon.stub(closeButton, 'getBoundingClientRect').returns({ width: 0 });
+    try {
+      window.dispatchEvent(new CustomEvent('resize'));
+    } finally {
+      closeButton.getBoundingClientRect.restore();
```

Comment:
Rather then doing this in a finally, you can use the sinon sandbox and restore in in an afterEach. Then you wouldn't need both of these. Or is there something else about the `try` that's beneficial?